frontend/appflowy_flutter/lib/plugins/database_view/application/database_controller.dart

```diff
@@ -78,9 +80,11 @@ class DatabaseController {
   final String viewId;
   final DatabaseViewBackendService _databaseViewBackendSvc;
   final FieldController fieldController;
+  late final SortController sortController;
```

Comment:
Would it be better if we use _fieldController, _sortController to make it private?

Reply:
I access all 3 of these in different parts. In the sort menu, filter menu, etc. By making these private, I'm assuming you want to abstract out these controllers' functionality, for example databaseController.sortController.addListener => databaseController.addSortListener. I don't mind making the abstraction, but I hope we're not adding unneeded complexity here

Reply:
Yep. Only expose the necessary internal state.